menas-web/ui/components/schema/schemaDetail.controller.js

```diff
@@ -388,7 +388,7 @@ sap.ui.define([
 
     checkRegistryIntegration: function () {
       jQuery.ajax({
-        url: "api/schema/features",
+        url: window.apiUrl + "schema/features",
```

Comment:
previous URLs start with / this one doesn't. Please align the URLs

Reply:
I thought it was comitted